app/Http/Controllers/PublicWikiController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\WikiSiteStats;
+use Illuminate\Http\Request;
+use App\Wiki;
+use App\Http\Resources\PublicWikiResource;
+
+class PublicWikiController extends Controller
+{
+    private static $defaultParams = [
+        'sort' => 'sitename',
+        'direction' => 'asc'
+    ];
+    /**
+     * Display a listing of the resource.
+     */
+    public function index(Request $request)
+    {
+        $request->validate([
+            'sort' => 'in:sitename,pages',
+            'direction' => 'in:desc,asc',
+            'is_featured' => 'boolean',
+            'is_active' => 'boolean',
+            'per_page' => 'numeric',
+            'page' => 'numeric'
+        ]);
+
+        $params = array_merge(self::$defaultParams, $request->input());
+        $query = Wiki::query();
+
+        if (array_key_exists('is_featured', $params)) {
+            $query = $query->where([
+                'is_featured' => boolval($params['is_featured'])
+            ]);
+        }
+
+        if (array_key_exists('is_active', $params) && $params['is_active']) {
+            $query = $query->whereRelation('wikiSiteStats', 'pages', '>', 0);
```

Comment:
The ticket currently say `> 1`, I need to double check if this is correct or needs to be changed.

https://phabricator.wikimedia.org/T339334

Reply:
The ticket is correct. I fixed this to be `> 1`